migrations/versions/12780a039923_populate_database.py

```diff
@@ -0,0 +1,82 @@
+"""populate database
+
+Revision ID: 12780a039923
+Revises: 67ac3b7d5c2f
+Create Date: 2021-09-14 14:44:50.802230
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '12780a039923'
+down_revision = '67ac3b7d5c2f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('profile',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=500), nullable=True),
+    sa.Column('first_name', sa.String(), nullable=True),
+    sa.Column('last_name', sa.String(), nullable=True),
+    sa.Column('avatar_url', sa.String(length=500), nullable=True),
+    sa.Column('bio_rendered', sa.String(), nullable=True),
+    sa.Column('interests', sa.String(), nullable=True),
+    sa.Column('before_rc', sa.String(), nullable=True),
+    sa.Column('during_rc', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('stint',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('profile_id', sa.Integer(), nullable=True),
+    sa.Column('type_stint', sa.String(), nullable=True),
+    sa.Column('start_date', sa.Date(), nullable=True),
+    sa.Column('end_date', sa.Date(), nullable=True),
```

Comment:
I think all of these are required, and should not be nullable.

Reply:
I made these nullable, thanks for catching this. The only exception I found was `end_date` since faculty don't have any end date.